Vagrantfile

```diff
@@ -61,6 +61,13 @@ Vagrant.configure("2") do |config|
     end
   end
 
+  # Hyper-V configuration
+  config.vm.provider "hyperv" do |h|
+    h.enable_virtualization_extensions = true
+    h.linked_clone = true
+    config.vm.synced_folder ".", "/vagrant", disabled: true
+  end
+
   # Configure additional disk space
   config.vm.provision "provision-disk", type: "shell", path: "provision-disk.sh", privileged: true
```

Comment:
also this needs to be commented out - as there's no shared mount these failure to work. 
Not sure why  though - above my Linux Sysadmin level

Reply:
It would be nice to get the extra storage working with hyperv otherwise things can get a bit tight for space sometimes... maybe it's just a case of moving the add disk space bits out of the virtualbox provider? Unfortunately I can't try it :(